.github/workflows/cwag-workflow.yml

```diff
@@ -0,0 +1,369 @@
+          docker login "${DOCKER_REGISTRY}" -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
+          IMAGES="goradius|xwfm-integ-tests"
+          IMAGES_ARRAY=($(echo $IMAGES | tr "|" "\n"))
+          PROJECT=cwag
+          TAG=${GITHUB_SHA:0:8}
+          TAG_LATEST=false
+          function tag_and_push {
+            docker tag "$IMAGE_ID" "${DOCKER_REGISTRY}/$IMAGE:$1"
+            echo "Pushing ${DOCKER_REGISTRY}/$IMAGE:$1"
+            docker push "${DOCKER_REGISTRY}/$IMAGE:$1"
+          }
+          for IMAGE in "${IMAGES_ARRAY[@]}"; do
+            IMAGE_TOSEARCH=$IMAGE
+            if [ ! -z $PROJECT ]; then
+              IMAGE_TOSEARCH="${PROJECT}_${IMAGE}"
+            fi
+            IMAGE_ID=$(docker images "$IMAGE_TOSEARCH:latest" --format "{{.ID}}")
+            tag_and_push "$TAG"
+            if [ "$TAG_LATEST" = true ]; then
+              tag_and_push "latest"
+            fi
+          done
+          #      - magma_slack_notify
+
+########## CWAG-DEPLOY-LATEST ##########
+
+  cwag-deploy-latest:
+
+    needs: [cwag-deploy, xwfm-deploy-latest]
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: |
+          echo "MAGMA_ROOT=${PWD}" >> $GITHUB_ENV
+          echo "DOCKER_REGISTRY=cwf_" >> $GITHUB_ENV
+    - name: Build the docker-compose stack
+      run: |
+           docker-compose up -d
+    - name: Build CWAG containers
+      run: |
+            cd ${MAGMA_ROOT}/cwf/gateway/docker
+            docker-compose -f docker-compose.yml -f docker-compose.override.yml build --parallel
+    - name: tag-push-docker
+      run: |
+          if [ "${GITHUB_BASE_REF}" != "master" ]; then
+            echo "Push only happens for master branch"
+            exit 0
+          fi
+          DOCKER_REGISTRY="$DOCKER_FEG_REGISTRY"
+          DOCKER_USERNAME="$USERNAME"
+          DOCKER_PASSWORD="$PASSWORD"
+          docker login "${DOCKER_REGISTRY}" -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
+          IMAGES="gateway_python|gateway_pipelined|gateway_go"
+          IMAGES_ARRAY=($(echo $IMAGES | tr "|" "\n"))
+          PROJECT=cwf
+          TAG=${GITHUB_SHA:0:8}
+          TAG_LATEST=true
+          function tag_and_push {
+            docker tag "$IMAGE_ID" "${DOCKER_REGISTRY}/$IMAGE:$1"
+            echo "Pushing ${DOCKER_REGISTRY}/$IMAGE:$1"
+            docker push "${DOCKER_REGISTRY}/$IMAGE:$1"
+          }
+          for IMAGE in "${IMAGES_ARRAY[@]}"; do
+            IMAGE_TOSEARCH=$IMAGE
+            if [ ! -z $PROJECT ]; then
+              IMAGE_TOSEARCH="${PROJECT}_${IMAGE}"
+            fi
+            IMAGE_ID=$(docker images "$IMAGE_TOSEARCH:latest" --format "{{.ID}}")
+            tag_and_push "$TAG"
+            if [ "$TAG_LATEST" = true ]; then
+              tag_and_push "latest"
+            fi
+          done
+    - name: persist-githash-version - Set prefix
+      run: |
+          echo "file_prefix=cwag" >> $GITHUB_ENV
```

Comment:
It would be better to use cwag in the command below instead of defining a variable for it